test/global-table.test.ts

```diff
@@ -18,6 +18,46 @@ test('Create global table with partition key and default properties', () => {
   });
   const template = assertions.Template.fromStack(newstack);
   template.resourceCountIs('AWS::DynamoDB::GlobalTable', 1);
+  template.hasResourceProperties('AWS::DynamoDB::GlobalTable', {
+    AttributeDefinitions: [{
+      AttributeName: 'id',
+      AttributeType: 'S',
+    }],
+    BillingMode: 'PAY_PER_REQUEST',
+    KeySchema: [{
+      AttributeName: 'id',
+      KeyType: 'HASH',
+    }],
+    Replicas: [{
+      Region: 'us-east-1',
+    }],
+  });
+});
+
+test('Create global table with partition key and specified replica, tablename', () => {
+  const region = 'us-east-1';
+  const app = new App();
+  const newstack = new Stack(app, 'id', {
+    env: { region: region },
+  });
+  new GlobalTable(newstack, 'hello', {
+    partitionKey: {
+      name: 'id',
+      type: ddb.AttributeType.STRING,
+    },
+    replicas: [{
+      region: newstack.region,
+    },
+    {
+      region: 'us-west-1',
+    },
+    {
+      region: 'us-west-2',
+    }],
```

Comment:
You should have a test that includes the stack's region in the replicas array, and a test that excludes it. If you plan to support both scenarios, they both should work. (Another option would be to throw an error when a customer supplies the stack's region as a replication region.)